Frontend/ui-customer/src/Screens/ZooTicketBookingFlow/TicketBookingScreen/TicketInfoScreen/ScreenData.purs

```diff
@@ -13,22 +13,76 @@
+      -- selectedBookingInfo : {shortId : "", ticketPlaceId : "", ticketPlaceName : "", personId : "" , amount : 500.0, visitDate : "2023-10-23", status : Booked, services : [{ ticketServiceShortId : "", ticketServiceName : "VideoPhotography", amount : 100.0, status : "Pending", verificationCount : 0, expiryDate : Nothing,  prices : [{pricePerUnit: 2.0,numberOfUnits: 3,peopleCategoryName: "Adults"}, {pricePerUnit: 2.0,numberOfUnits: 2,peopleCategoryName: "Mobile"}]}, { ticketServiceShortId : "afdj;jkja", ticketServiceName : "Entrance", amount : 100.0, status : "Confirmed", verificationCount : 0, expiryDate : Nothing,  prices : [{pricePerUnit: 2.0,numberOfUnits: 3,peopleCategoryName: "Adults"}, {pricePerUnit: 2.0,numberOfUnits: 2,peopleCategoryName: "Mobile"}]}, { ticketServiceShortId : "afdjasdf ;a", ticketServiceName : "Aquarium", amount : 100.0, status : "Pending", verificationCount : 0, expiryDate : Nothing,  prices : [{pricePerUnit: 2.0,numberOfUnits: 3,peopleCategoryName: "Adults"}, {pricePerUnit: 2.0,numberOfUnits: 2,peopleCategoryName: "Mobile"}]}] }
+      selectedBookingInfo : {
+          shortId : "String",
+          ticketPlaceId : "ticketPlcaId",
+          ticketPlaceName : "ticketPlaceName",
+          personId : "personId",
+          amount : 66.0,
+          visitDate : "2023-11-30",
+          status : Pending,
+          services : [dummyServiceDetails, dummyServiceDetails2]
+      }
   }
   , props : {
-        activeListItem : { ticketServiceShortId : "afdjasdf ;a", ticketServiceName : "Videography", amount : 100.0, status : "Pending", verificationCount : 0, expiryDate : Nothing,  prices : [{pricePerUnit: 2.0,numberOfUnits: 3,attendeeType: "Adults"}, {pricePerUnit: 2.0,numberOfUnits: 2,attendeeType: "Mobile"}]}
+        activeListItem : dummyServiceDetails
       , activeIndex : 0
       , rightButtonDisable : false
       , leftButtonDisable : true
     }
-  }
\ No newline at end of file
+  }
+
+dummyServiceDetails :: TicketBookingServiceDetails
+dummyServiceDetails =
+  { amount : 55.0,
+    status : "dkfj;a",
+    verificationCount : 5,
+    expiryDate : Just "akfjk;asd",
+    ticketServiceName : "dkj;akl",
+    categories : [dummyCategory, dummyCategory],
+    ticketServiceShortId : "dkajf;a",
+    slot : Just "13:00:00"
+  }
+
+dummyServiceDetails2 :: TicketBookingServiceDetails
+dummyServiceDetails2 =
+  { amount : 55.0,
+    status : "dkfj;a",
+    verificationCount : 5,
+    expiryDate : Just "akfjk;asd",
+    ticketServiceName : "dkj;ak asdfasfd  afasd l",
+    categories : [dummyCategory, dummyCategory],
+    ticketServiceShortId : "dkajf;a asdfasd ",
+    slot : Just "13:00:00"
+  }
+
+dummyCategory :: TicketBookingCategoryDetails
```

Comment:
This function is there 3 time please remove duplicate functions

Reply:
done